test/data/testinit.js

```diff
@@ -1,12 +1,12 @@
 /* eslint no-multi-str: "off" */
 "use strict";
 
-var parentUrl = window.location.href.replace( /\/test\/.*$/, "/" ),
+var parentUrl = window.location.origin,
```

Comment:
Some versions of IE 11 didn't support `location.origin`. I'm not sure if this is still an issue but I remember this pain...

Reply:
What do you think of removing `parentUrl` altogether and using absolute URLs? It works for selenium, browserstack, and jsdom and might be more future proof when it comes time to try this on 3.x. I think it was only necessary on testswarm?

I'll do a test run with it, but we can make changes if you have another reason for including the domain.

Reply:
Seems to work. Let me know if there's an issue with this.

Reply:
This may have been required because of local PHP testing (also used by TestSwarm) & Karma using different URLs. Perhaps irrelevant with your changes.